lib/cretonne/src/isa/encoding.rs

```diff
@@ -76,3 +77,5 @@ impl fmt::Display for DisplayEncoding {
         }
     }
 }
+
+impl PrimaryEntityData for Encoding {}
```

Comment:
This should not be needed. Use `EntityMap::ensure()` instead. See comment below.